gulpfile.js

```diff
@@ -18,14 +18,14 @@ const stream = require('webpack-stream');
 const inject = require('gulp-inject');
 const postcss = require('gulp-postcss');
 const sass = require('gulp-sass');
- 
-sass.compiler = require('node-sass')
 
+sass.compiler = require('node-sass');
 
+var config;
```

Comment:
Why not `let`?

Reply:
No particular reason :p

I made the change in a new commit.